LLama/Extensions/DictionaryExtensions.cs

```diff
@@ -10,7 +10,7 @@ internal static class DictionaryExtensions
             return GetValueOrDefaultImpl(dictionary, key, defaultValue);
         }
 #elif !NET6_0_OR_GREATER && !NETSTANDARD2_1_OR_GREATER
-    #error Target framework not supported!
+#error Target framework not supported!
```

Comment:
I prefer the old syntax - for the same reason you indent any other if statement!